compose/compose-file/index.md

```diff
@@ -691,6 +699,23 @@ If you have further questions, please refer to the discussion on the GitHub
 issue [docker/compose/4513](https://github.com/docker/compose/issues/4513){: target="_blank" class="_"}.
 {: .important}
 
+##### Generic Resources
+
+Generic Resources are a way to select the kind of nodes your task can land on.
+
+In a swarm cluster, nodes can advertise Generic resources as Discrete values or
+as Named values (e.g: SSD=3 or GPU=UID1, GPU=UID2, ...).
```

Comment:
please use the construction "..., such as...." instead of parentheses and the Latin abbreviation `.e.g.`